components/tooltip/test/tooltip.test.js

```diff
@@ -203,6 +204,26 @@ describe('d2l-tooltip', () => {
 			await oneEvent(tooltipFixture, 'd2l-tooltip-show');
 			expect(dynamicTooltip.showing).to.be.true;
 		});
+
+		it('should show if added to a target with a nested element that already has focus', async() => {
+
+			const targetTag = defineCE(class extends LitElement {
+				render() {return html`<button>nested target</button>`; }
+			});
+			const elem = await fixture(`<div><${targetTag} id="target"></${targetTag}></div>`);
+			const target = elem.querySelector(targetTag);
+
+			await focusElem(target.shadowRoot.querySelector('button'));
+
+			const tooltip = document.createElement('d2l-tooltip');
+			tooltip.announced = true;
```

Comment:
I used `announced` here to suppress the console warnings about the tooltip not being associated with an interactive element. I could have also set `for` to `target` and added `role` and `tabindex` to the target. 🤷